web/ui/react-app/src/pages/graph/ExpressionInput.tsx

```diff
@@ -215,12 +215,10 @@ class ExpressionInput extends Component<ExpressionInputProps, ExpressionInputSta
                   value={value}
                 />
                 <InputGroupAddon addonType="append">
-                  <Button className="btn-light border" title="Open metrics explorer" onClick={this.openMetricsExplorer}>
+                  <Button title="Open metrics explorer" onClick={this.openMetricsExplorer}>
```

Comment:
Due to the removal of `btn-light`, this button is now always dark, even in light mode.

Reply:
This one is a bit tricky since there are no border or other visual separators for buttons in a button group

https://getbootstrap.com/docs/4.6/components/button-group/

Which means that when you have a button group with buttons of different colour it tends to look a bit "raw", at least to my eye. I'll try to add some custom style to it.